### ISSUE TITLE
How can I send an image data when requesting an Execute session?

### ISSUE THREAD

**Issue by gzxy-0102** [kibae/onnxruntime-server#54]
How can I send an image data when requesting an Execute session? If I send a numpy array of images, there will be a body limit error

**Comment by kibae** [comment]
Hi, @gzxy-0102
I would like to know the following information to troubleshoot the problem.
- What is the size of your numpy array?
- Which protocol did you use, HTTP/HTTPS or TCP?
- What did the error message say?

**Comment by gzxy-0102** [comment]
> Hi, @gzxy-0102 I would like to know the following information to troubleshoot the problem.
> 
> * What is the size of your numpy array?
> * Which protocol did you use, HTTP/HTTPS or TCP?
> * What did the error message say?

Hi, @kibae 
I didn't record the size of numpy, but I can provide a picture
I am using the HTTP protocol
The error message is that the body has exceeded the limit

![mmexport1705047066507](https://github.com/kibae/onnxruntime-server/assets/8954610/ce722478-f58d-4c35-8f75-ac59a631c61d)

**Comment by kibae** [comment]
@gzxy-0102
There was a payload size limit for HTTP/HTTPS requests. I resolved this issue by adding the `--request-payload-limit` option. It can also be set using the environment variable `ONNX_SERVER_REQUEST_PAYLOAD_LIMIT`. The unit is in bytes.
https://github.com/kibae/onnxruntime-server?tab=readme-ov-file#options

Coincidentally, onnxruntime has been updated to version 1.18.1. I have prepared a new Docker image accordingly. If you are using the Docker image, you can pass the environment variable with an option like `-e "ONNX_SERVER_REQUEST_PAYLOAD_LIMIT=xxxxxxxxxxxxxxx"`.

Please let me know if you encounter any issues. Thank you.

**Comment by gzxy-0102** [comment]
> @gzxy-0102 There was a payload size limit for HTTP/HTTPS requests. I resolved this issue by adding the `--request-payload-limit` option. It can also be set using the environment variable `ONNX_SERVER_REQUEST_PAYLOAD_LIMIT`. The unit is in bytes. https://github.com/kibae/onnxruntime-server?tab=readme-ov-file#options
> 
> Coincidentally, onnxruntime has been updated to version 1.18.1. I have prepared a new Docker image accordingly. If you are using the Docker image, you can pass the environment variable with an option like `-e "ONNX_SERVER_REQUEST_PAYLOAD_LIMIT=xxxxxxxxxxxxxxx"`.
> 
> Please let me know if you encounter any issues. Thank you.

@kibae Thank you so much. I whill try it later

**Comment by gzxy-0102** [comment]
Hi @kibae 
It can now send images, but when I requested the API to register the model, the request to register the model kept running

**Comment by gzxy-0102** [comment]
> Hi @kibae It can now send images, but when I requested the API to register the model, the request to register the model kept running

<img width="584" alt="微信图片_20240701124838" src="https://github.com/kibae/onnxruntime-server/assets/8954610/fce04990-1c76-4f24-bf67-23a930ba2757">

**Comment by kilievich-dmitriy-andreevich** [comment]
> > Hi @kibae It can now send images, but when I requested the API to register the model, the request to register the model kept running
> 
> <img alt="微信图片_20240701124838" width="584" src="https://private-user-images.githubusercontent.com/8954610/344546575-fce04990-1c76-4f24-bf67-23a930ba2757.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcwNjM2NzAsIm5iZiI6MTcyNzA2MzM3MCwicGF0aCI6Ii84OTU0NjEwLzM0NDU0NjU3NS1mY2UwNDk5MC0xYzc2LTRmMjQtYmY2Ny0yM2E5MzBiYTI3NTcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjNUMDM0OTMwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Njk4N2Q1MGMwYTQxMDBjYzNjZjZmNzAyZjJlYjE1ZmQ5ZDk2OWM5YmZhYmM2NGI2ZjBmYTQwNmVkMzVkMTVkYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.ayjrEeIQSoaC0ZZYyps0hug-hRhxU-zHU1fAD-NXM4U">

Hello! Could you please send me the code to send the image? I'm interested in its function of converting from image to JSON

**Comment by kilievich-dmitriy-andreevich** [comment]
> Hi @kibae It can now send images, but when I requested the API to register the model, the request to register the model kept running

Hello! Could you please send me the code to send the image? I'm interested in its function of converting from image to JSON

**Comment by kilievich-dmitriy-andreevich** [comment]
> > Hi @kibae It can now send images, but when I requested the API to register the model, the request to register the model kept running
> 
> <img alt="微信图片_20240701124838" width="584" src="https://private-user-images.githubusercontent.com/8954610/344546575-fce04990-1c76-4f24-bf67-23a930ba2757.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcwNjM2NzAsIm5iZiI6MTcyNzA2MzM3MCwicGF0aCI6Ii84OTU0NjEwLzM0NDU0NjU3NS1mY2UwNDk5MC0xYzc2LTRmMjQtYmY2Ny0yM2E5MzBiYTI3NTcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjNUMDM0OTMwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Njk4N2Q1MGMwYTQxMDBjYzNjZjZmNzAyZjJlYjE1ZmQ5ZDk2OWM5YmZhYmM2NGI2ZjBmYTQwNmVkMzVkMTVkYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.ayjrEeIQSoaC0ZZYyps0hug-hRhxU-zHU1fAD-NXM4U">

Yes, I have exactly the same error, in the end I just refused logging in docker volumes and just watch them through exec so that they don't spam endlessly
